frontend/www/js/omegaup/components/problem/Edit.vue

```diff
@@ -117,14 +117,13 @@
       <div class="tab-pane active" v-if="showTab === 'markdown'">
         <omegaup-problem-markdown
           v-bind:markdown-contents="markdownContents"
-          v-bind:markdown-preview="markdownPreview"
           v-bind:initial-language="data.statement.language"
           v-bind:markdown-type="'statements'"
           v-bind:alias="data.alias"
           v-bind:title="data.title"
           v-bind:source="data.source"
-          v-bind:username="username"
-          v-bind:name="name"
+          v-bind:username="data.problemsetter.username"
+          v-bind:name="data.problemsetter.name"
```

Comment:
como es legítimo el caso en el que `data.problemsetter` sea `undefined` (según los tipos, pero después de leer el código es deseable mantener esa propiedad), entonces puede haber una manera para evitar el error que aparecía en las pruebas porque es un caso que puede ocurrirle a la gente en producción?

algo así como:

```suggestion
          v-bind:username="data.problemsetter?.username"
          v-bind:name="data.problemsetter?.name"
```

o mejor aún:

```suggestion
          v-bind:problemsetter="data.problemsetter"
```

y permitir en `StatementEdit.vue` que el `problemsetter` sea opcional.

Reply:
Listo